dynatrace-oneagent-operator/questions.yml

```diff
@@ -0,0 +1,74 @@
+categories:
+- APM
+- Monitoring
+questions:
+
+#################### Platform setting (hidden) ####################
+- variable: platform
+  default: "kubernetes"
+  type: string
+  show_if: false
+
+#################### Agent Configuration (REQUIRED) ####################
+- variable: oneagent.apiUrl
+  label: "Dynatrace API URL"
+  description: "Dynatrace API URL including `/api` path at the end"
+  default: "https://ENVIRONMENTID.live.dynatrace.com/api"
+  type: string
+  required: true
+  group: "Agent Configuration (REQUIRED)"
+
+- variable: secret.apiToken
+  label: "Dynatrace API token"
+  description: "Your Dynatrace API token - You can generate this token in your Dynatrace environment"
+  default: ""
+  type: string
+  required: true
+  group: "Agent Configuration (REQUIRED)"
+
+- variable: secret.paasToken
+  label: "Dynatrace PaaS token"
+  description: "Your Dynatrace Platform as a Service token - You can generate this token in your Dynatrace environment"
+  default: ""
+  type: string
+  required: true
+  group: "Agent Configuration (REQUIRED)"
+
+#################### Advanced Agent Configuration (OPTIONAL) ####################
+- variable: show_advanced_config
+  label: "Show advanced configuration"
+  description: "Show advanced configuration options for the Dynatrace OneAgent Operator"
+  default: false
+  type: boolean
+  group: "Advanced Agent Configuration (OPTIONAL)"
+
+- variable: operator.image
+  label: "Custom Operator image location"
+  description: "The location from where to grab the Dynatrace OneAgent operator image - default is quay.io/dynatrace/dynatrace-oneagent-operator"
+  default: ""
+  type: string
+  group: "Advanced Agent Configuration (OPTIONAL)"
+  show_if: "show_advanced_config=true"
+
+- variable: oneagent.image
+  label: "Custom OneAgent image location"
+  description: "The location from where to grab the Dynatrace OneAgent image - default for Kubernetes is docker.io/dynatrace/oneagent - default for OpenShift is registry.connect.redhat.com/dynatrace/oneagent"
```

Comment:
The default for OpenShift doesn't matter, right? Since you can't change the platform either way.